tools/test/integration_test.py

```diff
@@ -192,3 +200,12 @@ def test_bad_click_host(self, settings):
         assert len(tiles) == 2
         names = map(lambda tile: tile.get("name").lower(), tiles)
         assert list(names) == ["acme", "dunder mifflin"]
+
+    def test_aatimeout(self, settings):
```

Comment:
It seems like we don't actually run this test suite in CI?

Reply:
no, we don't. There's a comment above noting that I don't know how to best test this using the current integration tests, but created this local test to verify that it's working.